sdk/eventhub/ci.mgmt.yml

```diff
@@ -24,7 +24,10 @@ pr:
     - sdk/eventhub/ci.mgmt.yml
     - sdk/eventhub/service.projects
     - sdk/eventhub/Azure.ResourceManager.EventHubs
+    - sdk/resources/
     - sdk/resourcemanager/
+    - sdk/network/
```

Comment:
Why did we decide to add these extra triggers for these pipelines?

Reply:
Hi @weshaggard , the reason is that the tests in `Azure.ResourceManager.EventHubs` have project reference to these three projects. Sometimes we might only update these three projects but forget to update the tests here, this will break our pipeline especially the [core ci](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1663192&view=logs&jobId=e55623aa-2fbf-55e5-4c09-cf8db4d32009&j=e55623aa-2fbf-55e5-4c09-cf8db4d32009&t=b5ea0b4b-66b0-538b-2885-a23fc1bb1986). So, we add the extra triggers to ensure that we will not miss anything.